python/ray/serve/storage/kv_store.py

```diff
@@ -0,0 +1,251 @@
+import sqlite3
+
+import boto3
+import ray.experimental.internal_kv as ray_kv
+from botocore.exceptions import ClientError
+from ray.serve.storage.kv_store_base import KVStoreBase
+from ray.serve.utils import logger
+
+
+def get_storage_key(self, key):
```

Comment:
type annotate

Reply:
also from snapshot_head, it's called with multiple argument `get_storage_key(SERVE_CONTROLLER_NAME, SERVE_SNAPSHOT_KEY)` so it might error there?

Reply:
ah good call i forgot to change this while rebasing